packages/core/package.json

```diff
@@ -3,5 +3,10 @@
   "version": "0.0.1",
   "dependencies": {},
   "type": "commonjs",
-  "main": "./index.cjs"
+  "main": "./index.cjs",
```

Comment:
I guess I missed this before... Why is this a CJS package, when all our other packages are ESM? :open_mouth: 

I believe the `build` and `package.json` configuration should be exactly the same for `core` as for our other packages.

Reply:
will create a ticket for tooling